app/forms/questionnaire_form.py

```diff
@@ -255,7 +255,7 @@ def _has_min_and_max_single_dates(date_from, date_to):
 
     @staticmethod
     def _get_offset_value(period_object):
-        now = datetime.now()
+        now = datetime.now(tz=timezone.utc)
```

Comment:
Need to check whether this was done on purpose so that when do offset it uses system locale. We prob want to pass London timezone in if that is the case. For routing offset it uses `utc` but it also questions whether it should. Worth checking with Donna.

Reply:
I changed it to London, BST timezone as the offset needs to be at least one hour (one timezone) since this method returns minimum offset.

Reply:
I don't follow. This will only offset if we have specified one.

Reply:
I confirmed with Donna that we are supposed to use utc time as this is too much of an edge case.